### PR TITLE
Edit readme content and language

### PR DESCRIPTION
**Pull request by Renny4Real**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # GovWifi Logging API
 
-Session logging for successful authentications
+The frontend uses this API to record each session with successful authentications.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
 
 ## Overview
 
 Also known as post-auth in Freeradius terms, this logs to the sessions table when a user has authenticated successfuly.
```
Comment (tommotaylor) [inline README.md]: During the RADIUS `post-auth` action a POST request is sent to this API containing session data. This API receives this request and saves it to a database.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,32 @@
 # GovWifi Logging API
 
-Session logging for successful authentications
+The GovWifi frontend uses this API to record each session with successful authentications.
```
Comment (emileswarts) [inline README.md]: It records unsuccessful authentications as well.

"The GovWifi frontend uses this API to record each authentication request.  It is stored in a database and this data is used for reporting and debugging"

File: README.md [hunk 3]
```diff
@@ -1,14 +1,32 @@
...
+  - [Running the tests](#running-the-tests)
+  - [Using the linter](#using-the-linter)
+  - [Serving the app locally](#serving-the-app-locally)
+  - [Deploying changes](#deploying-changes)
+- [Licence](#licence)
 
 ## Overview
 
-Also known as post-auth in Freeradius terms, this logs to the sessions table when a user has authenticated successfuly.
+Also known as `post-auth` in FreeRadius terms, this logs to the sessions table when a user has authenticated successfully.
```
Comment (emileswarts) [inline README.md]: Successfully or unsuccessfully

File: README.md [hunk 4]
```diff
@@ -17,7 +35,13 @@ This application is also responsible for sending statistics to the Performance P
 
 This is useful for debugging and populating last_login of the user.
 
-### Statistics sent over to the performance platform
+### Sinatra routes
+
+- `GET /healthcheck` - AWS ELB target group health checking
+- `GET /logging/post-auth/user/?:username?/mac/?:mac?/ap/?:called_station_id?/site/?:site_ip_address?/result/:authentication_result` - Persist a
```
Comment (emileswarts) [inline README.md]: Yeah this is a post request now

File: README.md [hunk 5]
```diff
@@ -1,14 +1,32 @@
 # GovWifi Logging API
 
-Session logging for successful authentications
+The GovWifi frontend uses this API to record each authentication request. It is stored in a database and this data is used for reporting and debugging.
+N.B. The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
```
Comment (kjdchapman) [inline README.md]: I'd add an extra line break here:
```suggestion

N.B. The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
```

File: README.md [hunk 6]
```diff
@@ -40,20 +64,17 @@ aws ecs run-task --cluster wifi-api-cluster --task-definition logging-api-task-w
...
 
 ### Running the tests
 
-You can run the tests and linter with the following commands:
-
 ```shell
 make test
+```
+
+### Using the linter
```
Comment (kjdchapman) [inline README.md]: Could do a similar thing here as we did for the `make` commands in https://github.com/alphagov/govwifi-authentication-api/pull/60 , think that was a good way to do it.

**Comment by kjdchapman** [comment]
A couple of small suggestions - happy to approve it once they've been added/rejected.

File: README.md [hunk 7]
```diff
@@ -17,7 +33,13 @@ This application is also responsible for sending statistics to the Performance P
 
 This is useful for debugging and populating last_login of the user.
 
-### Statistics sent over to the performance platform
+### Sinatra routes
+
+- `POST /healthcheck` - AWS ELB target group health checking
+- `POST /logging/post-auth/user/?:username?/mac/?:mac?/ap/?:called_station_id?/site/?:site_ip_address?/result/:authentication_result` - Persist a
```
Comment (emileswarts) [inline README.md]: Oh sorry this is:

GET /healthcheck
POST /logging/post-auth

params:
  :username
  :mac
  :called_station_id
  :site_ip_address
  :authentication_result

**Pull request merged**